include/gmsh.tcc

```diff
@@ -80,6 +81,7 @@ void GMSH<Tdim, Tvertices>::read_vertices(std::ifstream& file) {
       this->vertices_.insert(std::make_pair(vertid, vertex));
     }
   }
+<<<<<<< HEAD
```

Comment:
Check merge, this seems like there is a merge issue.

Reply:
actually I had it in the next commit already sometime ago